plugins/media/live_cluster/lib/model/LiveClusterMediaServerNode.php

```diff
@@ -110,4 +111,14 @@ public function getExplicitLiveUrl($liveUrl, LiveStreamEntry $entry)
 		}
 		return parent::getExplicitLiveUrl($liveUrl, $entry);
 	}
+
+	public function getAdditionalUrlParam(LiveStreamEntry $entry) : string
```

Comment:
having return declaration type wont work on low php versions

Reply:
Move return type to comment